discovery-provider/plugins/network-monitoring/src/db/models/user.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes, ForeignKey, InferAttributes, InferCreationAttributes, Model } from "sequelize";
```

Comment:
We actually never used those models since everything is done with raw sql queries. I've been keeping them up to date in case we do want to use them instead of the raw queries but they're not really needed

Reply:
ohyeah, good point, interesting, ok - maybe put a note in db/models/readme or smth to indicate that

Reply:
I can probably just rm them - If for some reason in the future they're needed, they won't be too hard to get back (we have version control after all)

Reply:
https://github.com/AudiusProject/audius-protocol/pull/3422